package/yast2-network.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Nov 25 09:36:43 UTC 2019 - Knut Anderssen <kanderssen@suse.com>
+
+- Added a special type for the "Unknown" interfaces omitting them
+  from the the interfaces list (bsc#1156285)
```

Comment:
"the the"

Reply:
Fixed, and added with --amend to not add an extra commit just for fixing the typo.